docs/equality.md

```diff
@@ -67,9 +67,9 @@ eqVector.equals(
 ## Compare arrays
 
 ```ts
-import { array, eq } from "fp-ts";
+import { array, string } from "fp-ts";
 
-const eqArrayOfStrings = array.getEq(eq.eqString);
+const eqArrayOfStrings = array.getEq<string>(string.Eq);
```

Comment:
I'm surprised that we need to provide the generic value here, doesn't this work without `<string>`? (I haven't tried it)

Reply:
Removed.